src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier c = new ColorSupplier();
+        return new Ball(c.getRandomColor(), new Random().nextInt(100));
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/63095184/e3b8a8c7-e112-4717-b037-10ecfefddd19)

Reply:
1. not fixed
2. random should be `private final` class-level field